components/filter/filter-dimension-set.js

```diff
@@ -31,6 +31,11 @@ class FilterDimensionSet extends LitElement {
 			 * @type {boolean}
 			 */
 			loading: { type: Boolean },
+			/**
+			 * Wether the dimension has more values to load
```

Comment:
```suggestion
			 * Whether the dimension has more values to load. Must be used with selected-first and search-type="manual"
```
Could also be done as part of the documentation story if you're trying to get this merged before branching!